Marlin/src/libs/buzzer.h

```diff
@@ -84,9 +84,9 @@
 
     public:
       /**
-       * @brief Class constructor
+       * @brief Init Buzzer
        */
-      Buzzer() {
+      static inline void init() {
```

Comment:
`inline` is assumed when defining functions inside the class declaration. It doesn't hurt anything, but it not necessary.

Reply:
I just followed that is there.